WcaOnRails/config/routes.rb

```diff
@@ -51,6 +53,9 @@
   patch 'competitions/:competition_id/report' => 'delegate_reports#update'
 
   get 'competitions/:id/edit/admin' => 'competitions#admin_edit', as: :admin_edit_competition
+  get 'competitions/:id/payment_setup' => 'competitions#payment_setup', as: :competitions_payment_setup
+  get 'competitions/:id/revoke_stripe_access' => 'competitions#revoke_stripe_access', as: :competitions_revoke_stripe_access
```

Comment:
Am I correct in my understanding that this is a GET route that has a side effect? If so, this should be a POST.